lib/slide_hub/cloud/queue/response.rb

```diff
@@ -0,0 +1,33 @@
+module SlideHub
+  module Cloud
+    module Queue
+      class Message
+        attr_accessor :id
+        attr_accessor :body
+        attr_accessor :handle
+
+        def initialize(id, body, handle)
+          @id = id
+          @body = body
+          @handle = handle
+        end
+      end
+
+      class Response
+        attr :messages
```

Comment:
__[rubocop]__
* Style/Attr: Do not use `attr`. Use `attr_reader` instead. [:ref](https://github.com/bbatsov/ruby-style-guide#attr)